### PR TITLE
Update sample readme

### PR DESCRIPTION
**Pull request by sav007**
### What this does

Update readme with additional step to update git submodules

### CODE REVIEW

File: Sample Apps/Storefront/README.md [hunk 1]
```diff
@@ -8,10 +8,14 @@ The Mobile Buy SDK ships with a sample application that demonstrates how to buil
 
 The sample application is pre-configured with credentials to a test shop and requires no setup to build and run. You can also run the sample application with your own shop by following the steps below:
 
-1. Make sure that you have the **Mobile App** channel installed in your Shopify admin.
-2. From your Shopify admin, navigate to the **Mobile App** channel, and then copy your API key.
-3. Open the Sample Application project in `Sample App > Storefront > Storefront.xcodeproj`
-4. Open `Client.swift`, and then insert your API key and shop domain:
+1. Ensure that all submodules of Buy have alse been updated by running:
```
Comment (davidmuzi) [inline Sample Apps/Storefront/README.md]: add 'SDK' after 'Buy'

Comment (dbart01) [inline Sample Apps/Storefront/README.md]: - Buy -> `Buy`
- alse -> also

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-![Mobile Buy SDK](https://cloud.githubusercontent.com/assets/5244861/26374751/6895a582-3fd4-11e7-80c4-2c1632262d66.png)
+﻿![Mobile Buy SDK](https://cloud.githubusercontent.com/assets/5244861/26374751/6895a582-3fd4-11e7-80c4-2c1632262d66.png)
```
Comment (sav007) [inline README.md]: Don't know why this show as changes, though it identical

**Comment by sav007** [comment]
@davidmuzi @dbart01 updated

**Pull request merged**